controllers/confidential_handler.go

```diff
@@ -105,7 +105,7 @@ func (r *KataConfigOpenShiftReconciler) handleConfidentialPeerPods(state Feature
 	} else {
 		peerpodsCMData = map[string]string{
 			"DISABLECVM": "true",
-			"INITDATA": defaultNonCCInitdata,
+			"INITDATA":   defaultNonCCInitdata,
```

Comment:
Heh this one has been a pain in my dev workflow that includes `make fmt`. Thanks ! :wink:

We should have CI to block ill-formated PRs.

Reply:
True. I think another option is to have pre-commit hooks.